### PR TITLE
Adds basic 1.21.3 support.

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Fixes issue with spawning boats (though boats are still buggy and will be fixed in 2.10).
Bumps tests to 1.21.3, fixes issue with cow health test.
Adds new variants to attribute types lang file, apparently all the registry fields were renamed. 1 new one was added.
Adds pale garden to biome lang entry.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/main/resources/lang/default.lang [hunk 1]
```diff
@@ -2235,7 +2237,40 @@ attribute types:
 	player.sneaking_speed: player sneaking speed, sneaking speed
 	player.submerged_mining_speed: player submerged mining speed, submerged mining speed
 	player.sweeping_damage_ratio: player sweeping damage ratio, sweeping damage ratio
-	zombie.spawn_reinforcements: zombie spawn reinforcements
+	zombie.spawn_reinforcements: zombie spawn reinforcementsgeneric.armor: generic armor, armor
```
Comment (Efnilite) [inline src/main/resources/lang/default.lang:2240]:
```suggestion
	zombie.spawn_reinforcements: zombie spawn reinforcements
	generic.armor: generic armor, armor
```

File: src/main/resources/lang/default.lang [hunk 2]
```diff
@@ -2235,7 +2237,40 @@ attribute types:
...
+	# 1.21.3 removes prefixes, adds armor and tempt range
+	armor: generic armor, armor
```
Comment (Efnilite) [inline src/main/resources/lang/default.lang:2242]: does this allow `armour`?

File: src/main/resources/lang/default.lang [hunk 3]
```diff
@@ -2235,7 +2237,40 @@ attribute types:
...
+	scale: generic scale, scale
+	step_height: generic step height, step height
+	tempt_range: generic tempt range, tempt range
+	water_movement_efficiency: generic water movement efficiency, water movement efficiency
+	block_break_speed: player block break speed, block break speed
+	block_interaction_range: player block interaction range, block interaction range
+	entity_interaction_range: player entity interaction range, entity interaction range
+	mining_efficiency: player mining efficiency, mining efficiency
+	sneaking_speed: player sneaking speed, sneaking speed
+	submerged_mining_speed: player submerged mining speed, submerged mining speed
```
Comment (Efnilite) [inline src/main/resources/lang/default.lang:2271]: maybe add underwater as an option? or should it be as close to the original as possible

**Review by Efnilite — CHANGES_REQUESTED** [review]

File: src/main/resources/lang/default.lang (hunk 2 above)
Comment (sovdeeth) [reply src/main/resources/lang/default.lang:2242]: this is just for 1.21.3, so this is out of scope

File: src/main/resources/lang/default.lang (hunk 3 above)
Comment (sovdeeth) [reply src/main/resources/lang/default.lang:2271]: this is just for 1.21.3, so this is out of scope

**Review by Efnilite — APPROVED** [review]

**Review by cheeezburga — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]
seems fine

**Review by Moderocky — APPROVED** [review]

**Review by UnderscoreTud — APPROVED** [review]

**Pull request merged**